From d30018da40f9a6ce037a2acf2ea46d0ad3a08e6a Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 13 Jan 2021 21:25:38 +0000
Subject: [PATCH] SQUASH: Revert: "overlays: Make the i2c-gpio overlay
 safe again"

This revert and its neighbour are opposites. When squashing, delete
the original commits as well.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/i2c-gpio-overlay.dts
@@ -11,6 +11,9 @@
 		target-path = "/";
 
 		__overlay__ {
+			#address-cells = <1>;
+			#size-cells = <0>;
+
 			i2c_gpio: i2c@0 {
 				reg = <0xffffffff>;
 				compatible = "i2c-gpio";
